docs/HowToDebugCrashWithAddr2lineTool.md

```diff
@@ -0,0 +1,116 @@
+# How to use Addr2Line tool for debugging
+**addr2line** is a tool which comes with the Toolchain in Linux.
+It converts the given address into file names and line numbers.  
+Developers can debug the TizenRT ELF/AXF output file through **addr2line**.
+It is escpecially helpful which debugging a crash and can be used to find the 
+exact line number where the crash occured and also the function callstack
+which resulted in the crash.
+
+## Contents
+- [Prerequisites](#prerequisites)  
+- [How to use addr2line](#how-to-use-addr2line)
+- [How to debug a crash with addr2line](#how-to-debug-a-crash-with-addr2line)
+
+## Prerequisites
+To use addr2line, please enable below config.  
+
+In menuconfig, enable **CONFIG_DEBUG_SYMBOLS**.
+
+```
+menuconfig -> Debug Options -> Generate Debug Symbols
```

Comment:
For prerequisites, we'd better enable stack dump also.

Reply:
Done